### PR TITLE
Remote run

### PR DESCRIPTION
**Pull request by litzenberger**
Example to run on the local branch:

REMOTE='true' node local-add

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,16 @@ Create a pre-generated key:
 ```
 
 ### Run sub tests
-TODO:
+
+TESTCLASS:  Use env vairable TESTCLASS to run test just agianst that class.  Options of TestClass are define in the config.
```
Comment (mcollina) [inline README.md]:
```suggestion
Use env variable `TESTCLASS` to run test just against that class.  Options of `TESTCLASS` are defined in the config.
```

^ You should also add a link to the config.

File: README.md [hunk 2]
```diff
@@ -59,7 +59,16 @@ Create a pre-generated key:
 ```
 
 ### Run sub tests
-TODO:
+
```
Comment (mcollina) [inline README.md]:
```suggestion
#### `TESTCLASS`
```

File: README.md [hunk 3]
```diff
@@ -59,7 +59,16 @@ Create a pre-generated key:
...
+TESTCLASS:  Use env vairable TESTCLASS to run test just agianst that class.  Options of TestClass are define in the config.
+```bash
+> TESTCLASS="smallFile" node local-add
+```
+
```
Comment (mcollina) [inline README.md]:
```suggestion
#### SUBSTEST
```

File: README.md [hunk 4]
```diff
@@ -59,7 +59,16 @@ Create a pre-generated key:
 ```
 
 ### Run sub tests
```
Comment (mcollina) [inline README.md]: Can you please add a quick description of the difference of a testclass vs a subtest?

File: tests/config/index.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+{"subTest": "populated-repo","testClass": ["largeFile","smallFile"] }],
+"localExtract": [{"subTest": "empty-repo","testClass": ["largeFile","smallFile"] },
+{"subTest": "populated-repo","testClass": ["largeFile","smallFile"] }],
+"localTransfer": [{"subTest": "empty-repo","testClass": ["largeFile","smallFile"] },
+{"subTest": "populated-repo","testClass": ["largeFile","smallfile"] }
 ],
-"multiPeerTransfer": [{"subtest": "empty-repo","testClass": ["largefile","smallfile"] },
-{"subtest": "populated-repo","testClass": ["largefile","smallfile"] }
+"multiPeerTransfer": [{"subTest": "empty-repo","testClass": ["largeFile","smallFile"] },
+{"subTest": "populated-repo","testClass": ["largeFile","smallFile"] }
```
Comment (mcollina) [inline tests/config/index.json]: Can you jsonlint this file?

Comment (litzenberger) [reply tests/config/index.json]: yeah that file doesn't look update/

File: tests/lib/node-factory.js [hunk 6]
```diff
@@ -2,10 +2,15 @@
 
 const createNode = require(`./create-node`)
 const IPFS = require('ipfs')
+const REMOTE = process.env.REMOTE || false
+const remoteIPFS = require('../../../js-ipfs')
```
Comment (Elexy) [inline tests/lib/node-factory.js]: this aligns nicely with https://github.com/ipfs/benchmarks/pull/32

File: README.md (hunk 4 above)
Comment (litzenberger) [reply README.md]: We are removing Testcalss and Subtests.

**Pull request merged**